src/components/GenderViolenceMain/index.jsx

```diff
@@ -13,19 +16,18 @@ export default function GenderViolenceMain() {
           md={{ span: 6, order: "first" }}
           xs={{ span: 12, order: "last" }}
         >
-          <h1>
-            You are not alone
-            <br />
-            Safe Place is
-            <span> HERE</span>
-          </h1>
-          <p>
-            Safe Place provides access to immediate help and support for victims
-            of violence. We offer a shelter, psychological help, lawyers and
-            doctors.
-          </p>
+          <h1
+            dangerouslySetInnerHTML={{
+              __html: t("home.genderViolenceMain.header", {
+                break: "<br />",
+                span: "<span>HERE</span>",
```

Comment:
is this not translated?

Reply:
span is not needed in arabic translation because it is used for styling only in english